src/main/java/ai/knowly/langtorch/connector/pdf/PDFConnectorOptionNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package ai.knowly.langtorch.connector.pdf;
+
+public class PDFConnectorOptionNotFoundException extends RuntimeException {
+    public PDFConnectorOptionNotFoundException() {
```

Comment:
put exception into the constructor and pass it to the parent constructor otherwise we wont know what happens but only showing it's an exception